### PR TITLE
DO-244(19 May)

### PR DESCRIPTION
**Pull request by Palakagrawal28**

### CODE REVIEW

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx [hunk 1]
```diff
@@ -84,10 +84,12 @@ class UpdatePassword extends React.Component {
 
     // FIXME: Use typography components.
     return (
-      <div className={'update-password__header'}>
-        <span className={'BackArrow'}>
-          {this.renderBackToHomeButton()}
-        </span>
+      <div className={'update'}>
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx:87]:
```suggestion
      <div className={'update-password__header-container'}>
```

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss [hunk 2]
```diff
@@ -25,9 +25,9 @@
     @media(max-width: 1024px) {
       width: 60px;
       height: 60px;
-      top: 50%;
+      top: 55%;
       left: 39px;
-      position: absolute;
+      //position: absolute;
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss:30]: commented code!!

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss [hunk 3]
```diff
@@ -45,19 +45,20 @@
...
   padding-bottom: 50px;
   @media(max-width: 1024px) {
-    width: calc(70% - 60px);
+    width: 70%;
   }
 
   .update-password__form-container {
     width: 1024px;
-    margin: 0 auto;
+    //margin: 0 auto;
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss:53]: Here as well

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx (hunk 1 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx]: Done

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss (hunk 2 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss]: done

File: src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss (hunk 3 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/_UpdatePassword.scss]: Done

**Pull request merged**